.github/actions/build-platform-docker/action.yml

```diff
@@ -11,11 +11,12 @@ inputs:
     default: 'camunda/zeebe'
     required: true
   version:
-    description: 'The image version, e.g. SNAPSHOT, 8.1.0'
+    description: 'The image version, e.g. SNAPSHOT, 8.1.0. Can also accept a list of versions to create multiple tags for the same image.'
```

Comment:
🔧 Can we have an example like, is it comma separated, etc.?

Reply:
👍 
it cannot be comma separated, but array of strings or multiline strings like
```
version: |
  8.6.0
  latest
```
which I cannot put in the description string
I will put array example